src/scmrepo/git/lfs/client.py

```diff
@@ -272,3 +275,18 @@ def _as_atomic(to_info: str, create_parents: bool = False) -> Iterator[str]:
         raise
     else:
         shutil.move(tmp_file.name, to_info)
+
+
+class _ExponentialRetry(ExponentialRetry):
+    def get_timeout(
+        self, attempt: int, response: Optional[aiohttp.ClientResponse] = None
+    ) -> float:
+        if response is not None and response.status == HTTPStatus.TOO_MANY_REQUESTS:
+            if "Retry-After" in response.headers:
+                with suppress(ValueError):
+                    return float(response.headers["Retry-After"])
```

Comment:
Looks like `Retry-After` could be a timestamp too. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After.

Reply:
Never mind, let's leave it for now. Support for GitHub/GitLab may be good enough.